Client/Cliqz/Foundation/Environment/SettingsPrefs.swift

```diff
@@ -182,6 +185,30 @@ class SettingsPrefs {
         
     }
     
+    class func getLogTelemetryPref() -> Bool {
+        let defaultValue = false
+        if let logTelemetryPref = SettingsPrefs.getBoolPref(LogTelemetryPrefKey) {
+            return logTelemetryPref
+        }
+        return defaultValue
```

Comment:
can this be written in this way? `return SettingsPrefs.getBoolPref(LogTelemetryPrefKey) || false`

Reply:
Actually it could be 
`return SettingsPrefs.getBoolPref(LogTelemetryPrefKey) ?? false` 
but I think it is more readable this way